### DISCUSSION TITLE
Проблемы с отображением фона и аватара в изображении для приветственного сообщения

### DISCUSSION THREAD

**Discussion by rovius** [LacunaHub/.github#5] [Общее]
При заходе участника на сервер, если в приветствии включена отправка изображений, используется цвет фона + собственный фон по разрешенной ссылке и аватарка участника в слоях, фон и аватарка не показываются. В [ветке](https://discord.com/channels/740586549145763960/1219287781377839134/1219287781377839134) на сервере была догадка, что это из-за того, что пункт "цвет фона изображения" перекрывает собой все остальные (меняли цвет, прозрачность фона и вся плашка была именно этого цвета, фонового изображения под ней не было). На другом сервере, где настройка фона изначально не была включена - аватар работает. А удалить цвет фона полностью нет возможности - лишь выставить `rgba(0,0,0,0)`, что не помогает в данной ситуации. Ссылка используется из списка разрешённых, аватар участника - через заменитель.

**Comment by danliyev** [comment]
Добавлен крестик в поле выбора цвета

![image](https://github.com/LacunaHub/.github/assets/30476012/a897273c-87f6-4273-917c-b3ad53a6da54)
